packages/xdl/src/Webpack.ts

```diff
@@ -539,6 +539,22 @@ async function startNextJsAsync({
   await app.prepare();
 
   const server = express();
+
+  server.get('/expo-service-worker.js', (req, res) => {
```

Comment:
Will this work for locally testing prod builds? Some people use serve-cli to test before deploying.

Reply:
It will work with `expo start --no-dev --web`. However, I don't think it will work with `expo build:web` and then `yarn next start` (since then it's not using our Express server).

It will work with `now` with the `now.json` configuration in the docs.